app/realty/domain/project/selector.py

```diff
@@ -0,0 +1,48 @@
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+from django.db.models import Count, Prefetch
+
+from realty.domain.building.entities import BuildingEntity
+from realty.domain.project.entities import ProjectEntity
+from realty.models.building import Building
+from realty.models.project import Project
+
+
+class ProjectSelector:
+    @staticmethod
+    def get_all_projects():
+        projects = Project.objects.all()
+        return projects
+
+    @staticmethod
+    def get_project_detail(pk):
+        try:
+            project = (Project.objects.prefetch_related(
+                Prefetch('building_set', queryset=Building.objects.annotate(total_flats=Count('flat'))))
+                       .get(id=pk))
```

Comment:
Форматирование нужно поправить
```suggestion
            project = (
                Project.objects.prefetch_related(
                    Prefetch('building_set', queryset=Building.objects.annotate(total_flats=Count('flat')))
                )
                .get(id=pk)
            )
```